code/controllers/subsystem/init/more_init_stuff.dm

```diff
@@ -78,8 +78,8 @@ var/datum/subsystem/more_init/SSmore_init
 //		testing("Generating [S], Blood([species_blood])")
 		for(var/datum/organ/external/O in H.organs)
 			//testing("[O] part")
-			for(var/brute = 1 to 3)
-				for(var/burn = 1 to 3)
+			for(var/brute = 0 to 3)
```

Comment:
These two loops were supposed to generate icons for all combinations from `"00"` to `"33"` but there was an off-by-one error, which resulted in having to generate those 0 icons whenever they were needed.
Also, this proc only seems to generate icons for humans, vox and diona, for some reason. I'm focusing on humans, for now, though.